### PR TITLE
rel id in no longer stored in ConsumerBase

### PR DESCRIPTION
**Pull request by dstathis**

### CODE REVIEW

File: lib/charms/prometheus/v1/prometheus.py [hunk 1]
```diff
@@ -94,8 +93,7 @@ def remove_endpoint(self, address, port=80, rel_id=None):
                 :class:`PrometheusConsumer` has been instantiated in
                 `multi` mode.
         """
-        if rel_id is None:
-            rel_id = self.relation_id
+        rel_id = self.framework.model.get_relation(self._relation_name, rel_id).id
```
Comment (sed-i) [inline lib/charms/prometheus/v1/prometheus.py:96]: remove the 'rel_id` method argument?

File: lib/charms/prometheus/v1/prometheus.py [hunk 2]
```diff
@@ -69,8 +69,7 @@ def add_endpoint(self, address, port=80, rel_id=None):
...
```
Comment (sed-i) [inline lib/charms/prometheus/v1/prometheus.py:72]: remove the `rel_id` method argument?

File: lib/charms/prometheus/v1/prometheus.py (hunk 1 above)
Comment (sed-i) [inline lib/charms/prometheus/v1/prometheus.py:96]: reminds me the ren & stimpy episode where stimpy climbed into his own belly button

Comment (balbirthomas) [reply lib/charms/prometheus/v1/prometheus.py:96]: What are you trying to achieve here ? Please explain.  I do not see a commit message or a comment. Are you trying to fix a bug for instance ? What is the bug ?

**Comment by dstathis** [comment]
This was a false bug. Closing.

**Pull request closed without merging**